repository-setup.md

```diff
@@ -0,0 +1,8 @@
+# Repository Layout
+
+* Master is always protected.
+* Review where possible must have a second pair of eyes.
+* Squash merging is up to the repository primary maintainer
+  * We aim to have clean understandable updates, and good git history. Rather than hiding a mess under the carpet.
```

Comment:
I think you need to add an empty line here. If you see the [View](https://github.com/sous-chefs/meta/blob/71e30a9968fca7eb89a59fb67420babea6bf7683/repository-setup.md), the `# Example` heading is not interpreted correctly.